easybuild/easyconfigs/c/cutadapt/cutadapt-2.7-GCCcore-8.3.0-Python-3.7.4.eb

```diff
@@ -0,0 +1,48 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+# Modified by: Adam Huffman, Jonas Demeulemeester
+# The Francis Crick Institute
+# Modufied by: Albert Bogdanowicz
+# Institute of Biochemistry and Biophysics PAS
+
+easyblock = 'PythonBundle'
+
+name = 'cutadapt'
+version = '2.7'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://opensource.scilifelab.se/projects/cutadapt/'
+description = """Cutadapt finds and removes adapter sequences, primers, poly-A tails and
+ other types of unwanted sequence from your high-throughput sequencing reads."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+builddependencies = [('binutils', '2.32')]
+
+dependencies = [('Python', '3.7.4')]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('xopen', '0.8.4', {
+        'checksums': ['dcd8f5ef5da5564f514a990573a48a0c347ee1fdbb9b6374d31592819868f7ba'],
+    }),
+    ('dnaio', '0.4.1', {
+        'checksums': ['371a461bac0e821ff52f6235f0de4533ac73b0e990b470e9625486f2e6df2cd7'],
+    }),
+    (name, version, {
+        'checksums': ['070dec8d94b8bda72906c614b9e71bd61254a67a176dd17e5b57671edd567983'],
+    }),
+]
+
```

Comment:
no need for the `fix_python_shebang_for` and `sanity_check_commands`?

Reply:
yep, there is no need, the default `sanity_check_commands` already check the `cutadapt` module and the shebang is ok. Fixed :+1:

Reply:
@lexming Why was `sanity_check_commands` removed? There are no sanity check commands by default (only paths), and they were put in place for a good reason (see #9219)

Reply:
The `sanity_check_commands` was importing the python module with `python -c 'import cutadapt.seqio'`, which seemed unnecessary as the `cutadapt` module is already imported by default.

Reply:
That was added in #9219 to catch the missing `xopen`. `import cutadapt` works fine if `xopen` is missing, `import cutadapt.seqio` does not .

And it's better to leave the `--help` test in too, it's a good way to ensure the shebang is correct.

Reply:
and the previous `sanity_check_commands` were not testing the main module `cutadapt` as far as I remember.

Reply:
@lexming No, but that's done by default anyway when using `PythonPackage` (and not via `sanity_check_commands`, that's done separately).

Reply:
`cutadapt.seqio` does not exist in cutadapt v2. It should not have been removed from cutadapt v1 though. Fix in #9869